src/tmpfiles/tmpfiles.c

```diff
@@ -178,20 +181,25 @@ STATIC_DESTRUCTOR_REGISTER(arg_include_prefixes, freep);
 STATIC_DESTRUCTOR_REGISTER(arg_exclude_prefixes, freep);
 STATIC_DESTRUCTOR_REGISTER(arg_root, freep);
 
+static int specifier_host_name_isol(char specifier, const void *data, const void *userdata, char **ret);
 static int specifier_machine_id_safe(char specifier, const void *data, const void *userdata, char **ret);
+static int specifier_boot_id_isol(char specifier, const void *data, const void *userdata, char **ret);
+static int specifier_os_field_isol(char specifier, const void *data, const void *userdata, char **ret);
 static int specifier_directory(char specifier, const void *data, const void *userdata, char **ret);
+static int specifier_kernel_isol(char specifier, const void *data, const void *userdata, char **ret);
+static int specifier_arch_isol(char specifier, const void *data, const void *userdata, char **ret);
 
 static const Specifier specifier_table[] = {
         { 'm', specifier_machine_id_safe, NULL },
-        { 'b', specifier_boot_id,         NULL },
-        { 'H', specifier_host_name,       NULL },
-        { 'l', specifier_short_host_name, NULL },
-        { 'v', specifier_kernel_release,  NULL },
-        { 'a', specifier_architecture,    NULL },
-        { 'o', specifier_os_id,           NULL },
-        { 'w', specifier_os_version_id,   NULL },
-        { 'B', specifier_os_build_id,     NULL },
-        { 'W', specifier_os_variant_id,   NULL },
+        { 'b', specifier_boot_id_isol,    NULL },
```

Comment:
isol? that sounds like a weird suffix. What's that even supposed to mean? short for "isolated"?

Reply:
Yes, suggest a better name ? `rooted`?

Reply:
changed in new version